easybuild/easyconfigs/l/libgd/libgd-2.2.5-GCCcore-6.4.0.eb

```diff
@@ -1,7 +1,7 @@
 easyblock = 'ConfigureMake'
 
 name = 'libgd'
-version = '2.2.4'
+version = '2.2.5'
```

Comment:
@reedts You're now renaming the `libgd` 2.2.4 easyconfig to `2.2.5`, which means removing the `2.2.4` easyconfig...

Also, the checksum for `libgd` 2.2.5 needs to be fixed.

Reply:
@boegel I fixed the checksum but still struggling to get that version 2.2.4 out. Will removing it and readding it work (before I try and it still won't work)?

Reply:
@reedts Just re-adding it should do it (you can't remove it anymore, it's already gone since it was renamed to `2.2.5`).